docs/content/integrations/duckdb/using-duckdb-with-dagster.mdx

```diff
@@ -30,45 +29,203 @@ To complete this tutorial, you'll need:
 
 ---
 
-## Step 1: Configure the DuckDB I/O manager
+This tutorial is divided in two parts. Both parts will create the same assets, but how the data is stored in DuckDb will differ. In [Part 1](#part-1-using-the-duckdb-resource) you will use the DuckDB resource to manually execute SQL queries to create tables and interact with those tables. In [Part 2](#part-2-using-the-duckdb-io-manager) you will transfer the responsibility for generating the SQL to create and interact with tables to the DuckDB I/O manager. When writing your own assets, you may choose one or the other (or both) approaches depending on your storage requirements. See [When to use I/O managers](TODO-LINK) to learn more.
 
-The DuckDB I/O manager requires some configuration to connect to your database. You must provide a path where a DuckDB database will be created. Additionally, you can specify a `schema` where the DuckDB I/O manager will create tables.
+## Part 1: Using the DuckDB Resource
 
-```python file=/integrations/duckdb/tutorial/io_manager/configuration.py startafter=start_example endbefore=end_example
-from dagster_duckdb_pandas import DuckDBPandasIOManager
+
+### Step 1: Configure the DuckDB resource
+
+To use the DuckDB resource, you add it to your `Definitions` object. The DuckDB resource requires some configuration. You must set a path to a duckdb database as the `database` configuration value. If the database does not already exist, it will be created for you:
```

Comment:
```suggestion
To use the DuckDB resource, you'll need to add it to your `Definitions` object. The DuckDB resource requires some configuration. You must set a path to a DuckDB database as the `database` configuration value. If the database does not already exist, it will be created for you:
```